router/ranking.js

```diff
@@ -71,12 +55,12 @@ RankingInterface.prototype.getUserRank = async function(req, res) {
     }
 }
 
-RankingInterface.prototype.buildRankData = async function(rankWithScores) {
+RankingInterface.prototype.buildRankData = async function(rawRankData) {
```

Comment:
```suggestion
const buildRankData = (rawRankData) => {
```
Interface.prototype을 없애는 방향으로 가는 건 어떨까요? 저게 클래스 패턴, 객체 지향 프로그래밍에서 흔히 사용하는 방식인데 자바스크립트에서는 맞지 않은 방식인 것 같아요.
모듈화하는게 좀 더 자바스크립트 철학에 맞지 않을까요?

Reply:
저는 js알못이라서... 태준님께 발언권을 넘기겠습니다.

Reply:
ecs6이전에 자바스크립트에 Class라는 개념이 없어서 prototype으로 class 흉내를 내서 객체지향 코드를 작성했다고 알고있는데요~ prototype을 없애자는 말이 esc6의 class 형식으로 코드를 바꾸자는 뜻인가요?

Reply:
prototype이 태준님하고 같은 생각이예요 근데 class 형식을 따라가자는 것이 아니예요 그 반대예요
RankingInterface.prototype.buildRankData 이렇게 하면 RankingInterface의 원시 객체에 buildRankData속성을 추가하는 것이랑 같은데 이렇게 하면 다른 객체에서 buildRankData를 사용하고 싶을 때 [ranking.js].RankingInterface.buildRankData를 사용해야 하고 RankingInterface의 의미가 없죠 그냥 [ranking.js].buildRankData로 하는 게 어떨까 싶어요~